### DISCUSSION TITLE
Installing inline documentation for the Rust standard library

### DISCUSSION THREAD

**Discussion by pieterdd** [rust-lang/rust-analyzer#16593] [Q&A] [answered]
Hover documentation is working for me:
![image](https://github.com/rust-lang/rust-analyzer/assets/1407980/615edc4d-5ff1-4445-bf24-43e81e660bc5)

Oddly enough, I only get it for library code and my own code. `std::process::Command::new` does not get a tooltip, for example. Is there a package I should install separately that provides this documentation? If it makes a difference: I installed `rust` and `rust-doc` via Fedora's repos, so I did not get it from rustup.

**Comment by flodiebold** [comment, marked as answer]
You probably need to install `rust-src` as well.

**Reply by pieterdd** [reply]
Bingo! Thank you @flodiebold.
